components/inference-service/README.md

```diff
@@ -0,0 +1,106 @@
+# MONAI Inference Service
+
+[![License](https://img.shields.io/badge/license-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
+
+MONAI Inference Service(MIS) is a server that runs MONAI Application Packages (MAP)[https://github.com/Project-MONAI/monai-deploy/blob/main/guidelines/monai-application-package.md] in a Kubernetes cluster. It shares the same
+principles with [MONAI](https://github.com/Project-MONAI).
+
+## Features
+
+> _The codebase is currently under active development._
+
+- Register a MAP in the Helm Charts of MIS.
+- Upload inputs via a REST API request and make them available to the MAP container.
+- Provision resources for the MAP container.
+- Provide outputs of the MAP container to the client which made the request.
+
+## Installation
+
+MIS supports following OS with **GPU/CUDA** enabled.
+
+- Ubuntu
+
+### Building the MIS Container
+
+To build the MIS container, you can simply run:
+```bash
+./build.sh
+```
+
+To build the MIS container manually, you can run:
+```bash
+docker build -f dockerfile -t monai/inference-service:0.1 .
+```
+
+### Helm Chart Configuration
+Helm charts are located in the charts folder.
```

Comment:
Usually, there is a blank line after section (`#`) and it is a valid syntax of Markdown though many parser can recognize this.